cli/src/auth/oidc.rs

```diff
@@ -347,12 +347,16 @@ pub struct UserInfo {
     pub name: Option<String>,
 }
 
-impl Display for UserInfo {
-    fn fmt(&self, f: &mut std::fmt::Formatter<'_>) -> std::fmt::Result {
+impl UserInfo {
+    pub fn identity(&self) -> String {
+        let mut identity = String::new();
+
         if let Some(name) = &self.name {
-            write!(f, "{} ", name)?;
+            let _ = write!(&mut identity, "{} ", name);
         }
-        write!(f, "<{}>", self.email)
+        let _ = write!(&mut identity, "<{}>", self.email);
```

Comment:
Much simpler and more efficient to just match on name and have two different format calls.

Reply:
"more efficient" seems like a bold claim...

But I can't argue with simpler